parliament_challenge.py

```diff
@@ -0,0 +1,159 @@
+import requests
+import waitress
+import flask
+from flask import Flask, request, json
+
+
+def get_speeches_api():
+    """
+    Use speech's api to get speech resources.
+    Speech's api gets latest speeches and size determines how many resources to get.
+    Loop through dict to return all values in the "anforande" key.
+    These values include the keys and values of the speeches.
+    """
+
+    domain = 'http://data.riksdagen.se'
+    speeches = 'anforandelista'
+    size = 10
+    format_type = 'json'
+    response = requests.get('{}/{}/?anftyp=Nej&sz={}&utformat={}'.format(domain, speeches, size, format_type))
+    if response.status_code == 200:
+        data = response.json()
+        for k, v in data.items():
+            anforande = v["anforande"]
+            return anforande
+
+
+def create_speeches_dict():
+    """
+    Use the get_speeches_api() function to create a dict of the relevant items.
+    """
+
+    speeches_data = get_speeches_api()
+    speeches_list = []
+    for speeches in speeches_data:
+        anforande_id = [v for k, v in speeches.items() if k == "anforande_id"]
```

Comment:
Why do you need a for loop here to get the value of `anforande_id`?

Reply:
Why you're making it so complicated? Make it simple

Reply:
The same for all keys! Think again how to make it simpler. All you're supposed to do is get the value for a key in a given dict and you know the key. so think again how to do it.

Reply:
updated